ewcfg.py

```diff
@@ -3285,6 +3303,9 @@ def wef_dclaw(ctn = None):
 		ctn.slimes_damage = int(ctn.slimes_damage * 1.5)
 		ctn.slimes_spent *= 1
 
+	dmg = ctn.slimes_damage
+	ctn.bystander_damage = int(dmg * 0.5)
```

Comment:
meaty already did something with the dragon claw. please check for compatibility

Reply:
I just added those lines of code, meaty added the same thing aswell in different parts of the weapon function, it will work fine as I removed one of those instances when resolving the mege. Even if it got pushed together it would just be a redundancy. If I made a mistake dm me on discord.